tests/jax/test_fused_attn.py

```diff
@@ -717,12 +762,23 @@ def _test_forward(
         is_training,
         qkv_layout,
         bias_shape,
+        swa,
     ):
         """
         Test forward with parameterized configs
         This test is not intended to run automatically during CI as it is time-consuming
         It is kept for development and debugging
         """
+        window_size_left = -1
+        window_size_right = -1
+        if swa:
+            # Use a small window size to avoid long running time
+            window_size = check_set_window_size(attn_mask_type, (random.randint(0, s_kv // 10), 0))
```

Comment:
Will `random.randint` be seeded or generate the same sequence?

Reply:
New commit: following the same practice in `_setup_inputs()` this test file, use a fixed seed for reproducibility.